rococo-parachains/shell-runtime/src/lib.rs

```diff
@@ -389,14 +287,7 @@ pub type SignedBlock = generic::SignedBlock<Block>;
 /// BlockId type as expected by this runtime.
 pub type BlockId = generic::BlockId<Block>;
 /// The SignedExtension to the basic transaction logic.
-pub type SignedExtra = (
-	frame_system::CheckSpecVersion<Runtime>,
-	frame_system::CheckGenesis<Runtime>,
-	frame_system::CheckEra<Runtime>,
-	frame_system::CheckNonce<Runtime>,
-	frame_system::CheckWeight<Runtime>,
-	pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
-);
+pub type SignedExtra = DisallowSigned;
```

Comment:
I'd say it is less error-prone in the future to leave the system extensions and just prepend the list with the new barrier.

Reply:
why reference intentially unused logic?

Reply:
fair enough 👍